src/display/annotation_layer.js

```diff
@@ -601,6 +601,23 @@ class AnnotationElement {
       triggers.classList.add("highlightArea");
     }
   }
+
+  _editOnDoubleClick() {
+    if (!this.linkService.eventBus) {
+      return;
+    }
+    const {
+      annotationEditorType: mode,
+      data: { id: editId },
+    } = this;
+    this.container.addEventListener("dblclick", () => {
+      this.linkService.eventBus.dispatch("switchannotationeditormode", {
```

Comment:
Similar to the rest of this file, I think that we can just do the following.
```suggestion
    const {
      annotationEditorType: mode,
      data: { id: editId },
    } = this;
    this.container.addEventListener("dblclick", () => {
      this.linkService.eventBus?.dispatch("switchannotationeditormode", {
```